src/main/scala/io/iohk/ethereum/rlp/RLPImplicits.scala

```diff
@@ -107,4 +107,15 @@ object RLPImplicits {
       }
   }
 
+  implicit def optionEnc[T](implicit enc: RLPEncoder[T]): RLPEncoder[Option[T]] = {
+    case None => RLPList()
+    case Some(value) => RLPList(enc.encode(value))
+  }
+
+  implicit def optionDec[T](implicit dec: RLPDecoder[T]): RLPDecoder[Option[T]] = {
```

Comment:
Maybe we should start using this for the optOut field eventually :thinking:

Reply:
I think we can't as we are using custom Boolean encoding here.

Reply:
Is it custom? I copied it from how it's done on the reverse field of GetBlockHeaders

Reply:
we don't have any default one, but IMHO default Boolean could be true/false not 1/0 ;)